tests/PSM.test.tsx

```diff
@@ -22,7 +22,7 @@ jest.mock('recharts', () => {
   return {
     ...OriginalModule,
     ResponsiveContainer: ({ children }: { children: typeof React.Children }) => (
-      <OriginalModule.ResponsiveContainer width={800} height={800}>
+      <OriginalModule.ResponsiveContainer width="100%" height={300}>
```

Comment:
lets keep the original widths. while that might result in warnings, its also gives a fixed non-zero width to the component being rendered. 
If we give a percentage instead, it results in the width of the component becoming 0 and the table component inside the page not being rendered. hence the update we are seeing in the snapshot.

Reply:
How come? I am seeing many components using widths in `%`.